README.md

```diff
@@ -34,6 +34,22 @@ where possible options include:
     -v, -version                    print version
 ```
 
+# How to use from Gradle
+
+Dukat will automatically run for npm dependencies by specifying the following in `build.gradle`:
+```
+kotlin.js.experimental.generateKotlinExternals=true
```

Comment:
This is only correct for Kotlin/JS Gradle plugin 1.3.x.
With 1.4 the property was renamed and you can now even configure per dependency whether to run Dukat on it or not.
But do you really think this belongs in the Dukat documentation?
I think this is more suited in the Kotlin/JS Gradle Plugin documentation and actually it is documented there: https://kotlinlang.org/docs/reference/js-external-declarations-with-dukat.html